contracts/VyperToken.vy

```diff
@@ -48,16 +48,13 @@ def decreaseAllowance(spender: address, subtractedValue: uint256) -> bool:
 
 @external
 def transfer(_to: address, _value: uint256) -> bool:
-    assert self.balanceOf[msg.sender] >= _value
     self.balanceOf[msg.sender] -= _value
     self.balanceOf[_to] += _value
     log Transfer(msg.sender, _to, _value)
     return True
 
 @external
 def transferFrom(_from: address, _to: address, _value: uint256) -> bool:
-    assert self.balanceOf[_from] >= _value
-    assert self.allowance[_from][msg.sender] >= _value
     self.allowance[_from][msg.sender] -= _value
```

Comment:
Can also add a `# dev: <message>` comment for the purposes of asserting the specific line fails during testing